### DISCUSSION TITLE
How do I use the code for this module

### DISCUSSION THREAD

**Discussion by Felipe634** [Poing-Studios/Godot-AdMob-Android-iOS#59] [Q&A]
Hello, I am a bit confused on how to implement the ad code with this module, also, do I need to do anything with the debug.a and release.a files?

**Comment by gumaciel** [comment]
Hello @Felipe634 , please read this tutorial according with your system:

Android: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS#android-v322
iOS: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS#ios-v33

You should first do this step i mentioned above, this is the Plugin of AdMob for Godot. Without this AdMob will never appear for you game.

These plugins have public functions that can be called on Godot, we already created an `API` to help you to use the functions of the Plugin, to install go here:

https://github.com/Poing-Studios/Godot-AdMob-Android-iOS#installation

The `API` is already configured to appear test ads, so if you did these two steps, then the plugin will work fine.

Here is how you can use the API: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS/wiki/AdMob-API#usage

e.g.: if you want to show banner just do that on any script of your project:

```gdscript
MobileAds.load_banner()
```

> do I need to do anything with the debug.a and release.a files?

The `.gdap` file is configured to use the `release.a` file, but you can change to `debug.a` to do a better testing, but generally you won't need to do that.

Btw i will convert this issue to discussion due is a doubt.
